tests/modules/test_securitydata.py

```diff
@@ -823,7 +826,8 @@ def test_stream_file_by_sha256_raises_py42_error_when_find_file_versions_returns
 
         with pytest.raises(Py42Error) as e:
             security_module.stream_file_by_sha256("shahash")
-            assert e.value.args[0] == PDS_EXCEPTION_MESSAGE.format("shahash")
+
+        assert e.value.args[0] == PDS_EXCEPTION_MESSAGE.format("shahash")
```

Comment:
I fixed some unrelated unit tests -- `with pytest.raises()` exits the `with` block as soon as an exception is raised so some assertions were not actually being checked

Reply:
Doesn't it still exit before execution this way? I thought that we have to do this in try/except to get it to work

Reply:
Nope, these lines definitely get hit, I ensured this with breakpoints and also by changing them to intentionally cause test failures.